application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -227,4 +219,4 @@ private static void handleErrors(@NotNull SlashCommandEvent event,
         return option == null ? null : Math.toIntExact(option.getAsLong());
 
     }
-}
+}
```

Comment:
no newline :disappointed:

Reply:
@Zabuzard gimme tutorial on how to add it as automatic action when commit, too lazy

Reply:
```suggestion
}

```